src/integration/appland/appMap.ts

```diff
@@ -0,0 +1,53 @@
+import { IncomingMessage } from 'http';
```

Comment:
Once this code is is accepted and merged, this can be refactored over to `appmap-js`. I have made the changes this way because I don't know how to develop scanner against a local branch of `appmap-js`.